src/Components/ExternalResult/ResultList.tsx

```diff
@@ -486,32 +359,17 @@ export default function ResultList() {
           </tbody>
         </table>
       </div>
-      {totalCount > RESULT_LIMIT && (
-        <div className="flex w-full pt-5 pb-1 items-center justify-center shadow sm:rounded-b-lg min-w-full bg-gray-50 border-t border-gray-200">
-          <Pagination
-            cPage={qParams.page}
-            defaultPerPage={RESULT_LIMIT}
-            data={{ totalCount }}
-            onChange={handlePagination}
-          />
-        </div>
-      )}
+      <Pagination totalCount={totalCount} />
       <CSVLink
         data={downloadFile}
         filename={`external-result--${now}.csv`}
         target="_blank"
         className="hidden"
         id={"downloadCSV"}
       />
-      <SlideOver show={showFilters} setShow={setShowFilters}>
+      <SlideOver {...advancedFilter}>
         <div className="bg-white min-h-screen p-4">
-          <ListFilter
-            filter={qParams}
-            onChange={applyFilter}
-            closeFilter={() => setShowFilters(false)}
-            dataList={lsgWardData}
-            local={local}
-          />
+          <ListFilter {...advancedFilter} dataList={lsgWardData} />
```

Comment:
```suggestion
          <ListFilter {...advancedFilter} dataList={lsgWardData} />
```

remove local in ListFilter also

Reply:
removed local filters in ListFilter